src/bucket/Bucket.cpp

```diff
@@ -24,6 +24,7 @@
 #include "util/TmpDir.h"
 #include "util/XDRStream.h"
 #include "xdrpp/message.h"
+#include <Tracy.hpp>
```

Comment:
should we have our own "util/Tracy.h" header that would ensure that we don't include anything tracy related if not build with `USE_TRACY`?